datamade/la-metro-dashboard
# Add list of Metro divisions to reduce overhead of dbinit

## Overview

See title. I made this change so that hopefully, scrape containers are no longer killed on staging.

### Checklist


## Testing Instructions

I tested locally, and also manually deployed to staging to test there. It works!!! Here's what I did:

 * (For local instance) Run the app locally as described in the README.
 * Turn on a scraping DAG, which will trigger it.
 * VIew the logs and confirm you see something like this:

```
*** Reading local file: /app/logs/windowed_bill_scrape_fri/scrape/2021-05-20T14:38:11.855780+00:00/1.log
[2021-05-20 14:38:21,133] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: windowed_bill_scrape_fri.scrape 2021-05-20T14:38:11.855780+00:00 [queued]>
[2021-05-20 14:38:21,153] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: windowed_bill_scrape_fri.scrape 2021-05-20T14:38:11.855780+00:00 [queued]>
[2021-05-20 14:38:21,156] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2021-05-20 14:38:21,157] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2021-05-20 14:38:21,158] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2021-05-20 14:38:21,176] {taskinstance.py:900} INFO - Executing <Task(BlackboxDockerOperator): scrape> on 2021-05-20T14:38:11.855780+00:00
[2021-05-20 14:38:21,183] {standard_task_runner.py:53} INFO - Started process 2289 to run task
[2021-05-20 14:38:21,282] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: windowed_bill_scrape_fri.scrape 2021-05-20T14:38:11.855780+00:00 [running]> 74a21c1975e5
[2021-05-20 14:38:21,319] {docker.py:289} INFO - Pulling docker image datamade/scrapers-us-municipal:staging
[2021-05-20 14:38:22,096] {docker.py:303} INFO - staging: Pulling from datamade/scrapers-us-municipal
[2021-05-20 14:38:22,099] {docker.py:298} INFO - Digest: sha256:af682302002513bf3e0789b8023859ab9540182bf6751a89e26589914589b06f
[2021-05-20 14:38:22,100] {docker.py:298} INFO - Status: Image is up to date for datamade/scrapers-us-municipal:staging
[2021-05-20 14:38:22,101] {docker.py:224} INFO - Starting docker container from image datamade/scrapers-us-municipal:staging
[2021-05-20 14:38:25,654] {docker.py:263} INFO - 05/20/2021 09:38:25 INFO pupa_settings.OCDClient: Raven is not configured (logging is disabled). Please see the documentation for more information.
[2021-05-20 14:38:26,276] {docker.py:263} INFO - Operations to perform:
  Apply all migrations: contenttypes, core, legislative, pupa
[2021-05-20 14:38:26,345] {docker.py:263} INFO - Running migrations:
  No migrations to apply.
[2021-05-20 14:38:26,628] {docker.py:263} INFO - 13 divisions found in the CSV, and 193493 already in the DB
[2021-05-20 14:38:31,223] {docker.py:263} INFO - The DB contains all CSV contents; no work to be done!
```

Handles #75
